orchestration/src/test/java/com/sap/ai/sdk/orchestration/OrchestrationUnitTest.java

```diff
@@ -273,8 +297,20 @@ void maskingPseudonymization() throws IOException {
     }
   }
 
-  @Test
-  void testErrorHandling() {
+  private static Runnable[] errorHandlingCalls() {
+    return new Runnable[] {
+      () -> client.chatCompletion(new OrchestrationPrompt(""), config),
+      () ->
+          client
+              .streamChatCompletion(new OrchestrationPrompt(""), config)
+              // the stream needs to be consumed to parse the response
+              .forEach(System.out::println)
```

Comment:
(Question)

What's the difference between the two calls?  
Looks like it doesn't affect assertions(?)

Reply:
`chatCompletion` <> `streamChatCompletion`